proto/babylon/incentive/query.proto

```diff
@@ -44,11 +44,26 @@ message QueryRewardGaugesRequest {
     string address = 1;
 }
 
+// RewardGaugesResponse is an object that stores rewards distributed to a BTC staking/timestamping stakeholder
+message RewardGaugesResponse {
+    // coins are coins that have been in the gauge
+    // Can have multiple coin denoms
+    repeated cosmos.base.v1beta1.Coin coins = 1 [
+        (gogoproto.nullable) = false,
+        (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+    ];
+    // withdrawn_coins are coins that have been withdrawn by the stakeholder already
+    repeated cosmos.base.v1beta1.Coin withdrawn_coins = 2 [
+        (gogoproto.nullable) = false,
+        (gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.Coins"
+    ];
```

Comment:
This Response type is exactly like the `RewardGauge` should we have two exactly structures just to split from the db used structure? @babylonlabs-io/core-dev

Reply:
I think this is the case, as the two structs might diverge in the future. If we reuse the same struct now, then it's made difficult if we want to make modifications on API or DB.